templates/hoover.nomad

```diff
@@ -196,6 +230,17 @@ job "hoover" {
               {{- end }}
             {{- end }}
 
+            {{- if service "tika" }}
+              {{- with service "tika" }}
+                {{- with index . 0 }}
+                  location ~ ^/(.*) {
```

Comment:
We don't need tika proxied by our nginx here, since all Snoop instances are getting it directly from `range service tika`. We do forward elasticsearch through here, but we don't need that one either.